File: src/main/java/net/maple/handlers/net/ClientDumpLogHandler.java
package net.maple.handlers.net;

import client.Client;
import net.maple.handlers.PacketHandler;
import util.HexTool;
import util.packet.PacketReader;

public class ClientDumpLogHandler extends PacketHandler {

    @Override
    public void handlePacket(PacketReader reader, Client c) {
        System.out.println("[ClientDumpLogHandler] " + HexTool.INSTANCE.toHex(reader.getData()));
    }
}

File: src/main/java/net/maple/handlers/net/ClientDumpLogHandler.kt
package net.maple.handlers.net

import client.Client
import net.maple.handlers.PacketHandler
import util.HexTool.toHex
import util.packet.PacketReader

class ClientDumpLogHandler : PacketHandler {

    override fun handlePacket(reader: PacketReader, c: Client) {
        println("[ClientDumpLogHandler] " + toHex(reader.data))
    }
}